src/Microsoft.Extensions.CodeGeneration/CodeGenCommand.cs

```diff
@@ -5,42 +5,42 @@
 using System.Collections.Generic;
 using System.Linq;
 using System.Reflection;
-using System.Threading.Tasks;
-using Microsoft.Extensions.DependencyInjection;
 
 namespace Microsoft.Extensions.CodeGeneration
 {
     public class CodeGenCommand
     {
+        private readonly ILogger _logger;
+        private readonly ICodeGeneratorLocator _locator;
+
```

Comment:
Its not needed for consumer usage. 
Should be safe to remove. The ServiceProvider didn't need to be public in the first place.